test/module/irohad/ametsuchi/block_query_test.cpp

```diff
@@ -212,3 +215,154 @@ TEST_F(BlockQueryTest, GetTransactionsWithInvalidTxAndValidTx) {
   });
   ASSERT_TRUE(wrapper.validate());
 }
+
+/**
+ * @given block store with 2 blocks totally containing 3 txs created by
+ * user1@test AND 1 tx created by user2@test
+ * @when get non-existent 1000 block
+ * @then nothing is returned
+ */
+TEST_F(BlockQueryTest, BlockQuery_GetBlocks_GetNonExistentBlock) {
```

Comment:
Maybe just `BlockQuery_GetNonExistentBlock`?

Reply:
Module_Method_TestCase

Reply:
Useless redundancy and it doesn't become more descriptive